### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by vivianha534**
**Created home page**
- created carousel 
- created vertical info cards
-  still need to make vertical cards mobile responsive, but will do after Joyce has finished with the graphic

### CODE REVIEW

File: src/components/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            }
+        </Carousel>
+    )
+}
+
+function Item(props)
+{
+    return (
+        <div className='opening-banner'>
+            <img className='banner-img' src={props.item.import ? props.item.importImg : props.item.imgSrc} />
```
Comment (tr89on) [inline src/components/Carousel.js:63]: Is there a reason to distinguish importImg and imgSrc?

Comment (vivianha534) [reply src/components/Carousel.js:71]: importImg is if we have the img stored in our assets folder, and imgSrc is if we pull off of the internet.

Comment (tr89on) [reply src/components/Carousel.js:71]: Yeah I get that, but is there a need for the distinction between the two? Both fields essentially serve the same function

Comment (vivianha534) [reply src/components/Carousel.js:71]: Tbh I wasn't sure, cause the hands image is stored in assets, but when I was making the component, I was pulling stock images from online, so I guess it just depends if we're gonna be using images from online when we actually deploy the site. I guess that would be something I need to ask Urna though.

Comment (tr89on) [reply src/components/Carousel.js:71]: Ok got it

File: src/components/VerticalCard.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 400,
+    marginRight: "50px",
+    marginBottom: "50px",
```
Comment (darren2hang) [inline src/components/VerticalCard.js:15]:
```suggestion
    margin: "5% 2%",
```

**Review by darren2hang — CHANGES_REQUESTED** [review]
Tiny nit. I think your removal of text-align: center from the App.css might make some of the styling messed up in the other parts of the app, but it won;t be hard to fix later.
Other than my one comment, I think its good.
I recommend just merging this into setUp.

**Review by darren2hang — APPROVED** [review]
LGTM!

**Comment by vivianha534** [comment]
merged into setUp

**Pull request closed without merging**